src/main/webapp/room.js

```diff
@@ -0,0 +1,19 @@
+let players = [];
+
+const createUser = (ev)=>{
+    ev.preventDefault();  //to stop the form submitting
```

Comment:
for good style, it's preferable to have a space after the `//`. 
Also personally, this is a very nitpicky thing, but the comment itself I prefer when its a complete sentence.
` // To stop the form from submitting.`